http/handler.go

```diff
@@ -133,7 +136,19 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	defer cancel()
 
-	req.Context = logging.ContextWithLoggable(req.Context, loggables.Uuid("requestId"))
+	reqId := loggables.Uuid("requestId")
+	req.Context = logging.ContextWithLoggable(req.Context, reqId)
+
+	// Create rootSpan for incoming request and tag
+	// All spans created after this point will be children of rootSpan
+	rootSpan := opentrace.StartSpan("ServeHTTP")
+	defer rootSpan.Finish()
+	ctx = opentrace.ContextWithSpan(ctx, rootSpan)
```

Comment:
I don't think we're actually doing anything with this context. I believe we should be replacing `req.Context`.